modules/base/tool.go

```diff
@@ -41,6 +41,14 @@ func ShortSha(sha1 string) string {
 	return TruncateString(sha1, 10)
 }
 
+// ShortSha256 takes 7-17 indexes of sha256 string as short sha256
+func ShortSha256(sha256 string) string {
+	if utf8.RuneCountInString(sha256) < 17 {
+		return sha256
+	}
+	return string([]rune(sha256)[7:17])
```

Comment:
Do not do that trick. Why `7-17`? Why not `8-18` .....

Reply:
0 is the start of the index？

Reply:
I mean it is fragile, you have made an assumption that there is a `sha256:` prefix, but the function name doesn't tell so.